scripts/gh-actions/set_eks_sysctl_config.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
```

Comment:
We could add `set -xe` to immediately exit on an error in the script
```suggestion
set -xe
```